docker-compose.yml

```diff
@@ -21,7 +21,7 @@ services:
   #
   faf-coturn:
     container_name: faf-coturn
-    image: bprodoehl/turnserver
+    image: geosearchef/coturn-lt-cred
```

Comment:
I don't know what this image contains, but shouldn't we move it to the faforever repo?

Reply:
we should (before it wasn't better :P , actually this image was terrible), I'm gonna move that to the faforever git and docker hub repos